src/test/scala/org/broadinstitute/dsde/workbench/sam/TestSupport.scala

```diff
@@ -122,8 +116,8 @@ object TestSupport extends TestSupport {
     SamDependencies(mockResourceService, policyEvaluatorService, new UserService(directoryDAO, googleExt, registrationDAO), new StatusService(directoryDAO, googleExt, dbRef), mockManagedGroupService, directoryDAO, policyDAO, googleExt)
   }
 
-  def genSamRoutes(samDependencies: SamDependencies)(implicit system: ActorSystem, materializer: Materializer): SamRoutes = new SamRoutes(samDependencies.resourceService, samDependencies.userService, samDependencies.statusService, samDependencies.managedGroupService, null, samDependencies.directoryDAO, samDependencies.policyEvaluatorService, LiquibaseConfig("", false))
-    with StandardUserInfoDirectives
+  def genSamRoutes(samDependencies: SamDependencies, uInfo: UserInfo)(implicit system: ActorSystem, materializer: Materializer): SamRoutes = new SamRoutes(samDependencies.resourceService, samDependencies.userService, samDependencies.statusService, samDependencies.managedGroupService, null, samDependencies.directoryDAO, samDependencies.policyEvaluatorService, LiquibaseConfig("", false))
+    with MockUserInfoDirectives
```

Comment:
This change is important. This means all tests that use this (pretty much all route testing) no longer use `StandardUserInfoDirectives` and headers to pass in users making requests. This is ok because there is now more comprehensive testing of `StandardUserInfoDirectives` and those other tests did not really care about the user anyway.